style/index.html

```diff
@@ -14,9 +14,9 @@
     <div class="background-opacity"></div>
     <h1 class="game-title">Wheel of Fortune!</h1>
     <div class="input-container">
-      <input class="player-inputs player1-input" type="text" name="player1" value="player 1">
-      <input class="player-inputs player2-input" type="text" name="player2" value="player 2">
-      <input class="player-inputs player3-input" type="text" name="player3" value="player 3">
+      <input class="player-inputs player1-input" type="text" name="player1" value="Player1">
+      <input class="player-inputs player2-input" type="text" name="player2" value="Player2">
+      <input class="player-inputs player3-input" type="text" name="player3" value="Player3">
       <button class="start-game-btn">READY!</button>
```

Comment:
quick fix to make it less janky when the player doesn't enter a input im assuming?